addons/website_sale_wishlist/views/website_sale_wishlist_template.xml

```diff
@@ -18,6 +18,23 @@
         </xpath>
     </template>
 
+    <template id="product_cart_lines" inherit_id="website_sale.cart_lines" name="Wishlist Button">
+        <div class="o_wsale_remove_from_cart" position="inside">
+            <span class="d-none d-md-inline-block ms-2">
+                | <a href='#' class="small o_add_wishlist js_delete_product ms-2"
```

Comment:
why need ms-2 here?

Reply:
We need to place the wishlist icon after remove, so in order to maintain space, I have added the margin.

Reply:
But for before space you added space before | and for after you added ms-2 it should be consistent